README.md

```diff
@@ -258,6 +272,17 @@ ansible-playbook migrate-db.yml                                    \
 
 It's safe to run migrations even if there are no changes - Django will detect that and simply leave the schema as is.
 
+#### Helper script for deployment process
+
+###### `build-and-deploy-to-dev.sh`
+This script is available on the remote `concent-builder` server.
+Before connect to server and run script, you must locally specify version of concent in the `containers/versions.yml` file and push it as commit to dev branch on the `concent-deployment` repository.
```

Comment:
Please reword this section:
``` markdown
This script provides a streamlined way to deploy a development version of the code to the `concent-dev` cluster.
The use case could be automated because it does not require as much flexibility as the usual deployment procedure.
The `concent-dev` cluster does not hold any production data and when something goes wrong it's fine to just remove everything and start from scratch.

The `configure.yml` playbook installs the script on the build server where it can use secrets necessary to access the cluster.
Then it can be run from that server by any user.

The script deploys the code from the `dev` branch in the `concent-deployment` repository.
Version of the `concent` repository is specified in `concent_version` in `containers/versions.yml`.
To deploy any other version you must change this value in your local working copy, push the commit to the repository and update the `dev` branch to point at it.
The value can be a commit ID, tag, branch.
```